.github/workflows/build-test-publish.yml

```diff
@@ -647,15 +647,16 @@ jobs:
         env: 
           TAG_NAME: ${{ github.ref }}
         run: |
-          # Tag is of format vX.Y.Z, cut the v off.
-          echo "NEW_VERSION=$(${TAG_NAME:1})" >> $GITHUB_OUTPUT
+          # Tag is of format refs/tags/vX.Y.Z, cut everything before X.Y.Z off.
+          echo "NEW_VERSION=$(${TAG_NAME:11})" >> $GITHUB_OUTPUT
```

Comment:
Can you please double check if this is correct? The CI in stable gave a command not found thing here.
I think it should be like I wrote in the discord without the double `$` and only the one with curly braces. 
`echo "NEW_VERSION=${TAG_NAME:11}" >> $GITHUB_OUTPUT`

Reply:
oh im dumb. the `$()` is a leftover from when there actually *was* a command in there. thanks for noticing